programs/client/Client.cpp

```diff
@@ -923,6 +923,7 @@ void Client::printHelpMessage(const OptionsDescription & options_description)
     std::cout << options_description.main_description.value() << "\n";
     std::cout << options_description.external_description.value() << "\n";
     std::cout << options_description.hosts_and_ports_description.value() << "\n";
+    std::cout << "All setting parameters are documented in detail at https://clickhouse.com/docs/ or you can also access them in `system.settings` table.\n\n";
```

Comment:
1. "in detail" is obsolete
2. the URL is too generic
3. "accessing" settings in system.settings is weird terminology.

--> `All settings are documented at

Reply:
I am getting the following output when I run `clickhouse-client` with `--help`:

```
~/repo/ch1/build/programs (reduce_help_messages =) $ chc --help
Main options:
  --help                              produce help message
  -V [ --version ]                    print version information and exit
  --version-clean                     print version in machine-readable format and exit
  -C [ --config-file ] arg            config-file path
  -q [ --query ] arg                  query; can be specified multiple times (--query "SELECT 1" --query "SELECT 2"...)
  --queries-file arg                  file path with queries to execute; multiple files can be specified (--queries-file file1 file2...)
  -n [ --multiquery ]                 If specified, multiple queries separated by semicolons can be listed after --query. For convenience,
                                      it is also possible to omit --query and pass the queries directly after --multiquery.
  -m [ --multiline ]                  If specified, allow multiline queries (do not send the query on Enter)
  -d [ --database ] arg               database
  --query_kind arg (=initial_query)   One of initial_query/secondary_query/no_query
  --query_id arg                      query_id
  --history_file arg                  path to history file
  --stage arg (=complete)             Request query processing up to specified stage: complete,fetch_columns,with_mergeable_state,with_mer
                                      geable_state_after_aggregation,with_mergeable_state_after_aggregation_and_limit
  --progress [=arg(=tty)] (=default)  Print progress of queries execution - to TTY: tty|on|1|true|yes; to STDERR non-interactive mode:
                                      err; OFF: off|0|false|no; DEFAULT - interactive to TTY, non-interactive is off
  -A [ --disable_suggestion ]         Disable loading suggestion data. Note that suggestion data is loaded asynchronously through a second
                                      connection to ClickHouse server. Also it is reasonable to disable suggestion if you want to paste a
                                      query with TAB characters. Shorthand option -A is for those who get used to mysql client.
  --wait_for_suggestions_to_load      Load suggestion data synchonously.
  -t [ --time ]                       print query execution time to stderr in non-interactive mode (for benchmarks)
  --echo                              in batch mode, print query before execution
  --verbose                           print query and other debugging info
```

There are two visual problems:
1. `--help` and `--verbose` are not next to each other
2. `--verbose` is a weird name, one could easily confuse it with "verbose" output for SELECT statements. What about `--verbose_help`?

Reply:
> `--verbose` is a weird name, one could easily confuse it with "verbose" output for SELECT statements.

this setting is now used for "verbose" output for SELECT statements and "verbose" output for --help. Wouldn't having two different `verbose` settings for different purposes be confusing?

Reply:
Yeah. Let's retain a single `--verbose` then (as currently implemented)